api/src/main/java/eu/darkbot/api/managers/RepairAPI.java

```diff
@@ -12,6 +12,11 @@
  */
 public interface RepairAPI extends API.Singleton {
 
+    /**
+     * Change death statistics to 0
+     */
+    void resetDeaths();
```

Comment:
in the future this (and getDeathAmount) probably should be part of stats api, but for now we'll let it stay here